### PR TITLE
[v2.8] Run integration and unit tests in GitHub Actions

### PR DESCRIPTION
**Pull request by rafaelbreno**
### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_

### CODE REVIEW

File: tests/v2/integration/catalogv2/rancher_managed_charts_test.go [hunk 1]
```diff
@@ -127,6 +127,7 @@ func (w *RancherManagedChartsTest) resetSettings() {
 }
 
 func TestRancherManagedChartsSuite(t *testing.T) {
+	t.Skip()
```
Comment (igomez06) [inline tests/v2/integration/catalogv2/rancher_managed_charts_test.go:130]: I don't think we need to skip the whole suite here
https://github.com/igomez06/rancher/commit/fb27c8240f86e37cd24cfde142641030153c35df#diff-fd1dcaa06d6248e12167940fa5128712872e50b2709831055cb2414c445157ecR64

I'm pretty sure the lower in the poll interval should fix it.

Comment (igomez06) [reply tests/v2/integration/catalogv2/rancher_managed_charts_test.go:130]: and only skip the `TestRancherManagedChartsSuite/TestServeIcons` test

**Review by briandowns — APPROVED** [review]

File: scripts/package [hunk 2]
```diff
@@ -31,7 +31,7 @@ docker build \
     --build-arg CATTLE_RANCHER_WEBHOOK_VERSION="${CATTLE_RANCHER_WEBHOOK_VERSION}" \
     --build-arg CATTLE_CSP_ADAPTER_MIN_VERSION="${CATTLE_CSP_ADAPTER_MIN_VERSION}" \
     --build-arg CATTLE_FLEET_VERSION="${CATTLE_FLEET_VERSION}" \
-    -t ${IMAGE} .
+    -t ${IMAGE} . --label org.opencontainers.image.created=2024-07-02T07:10:56Z --label org.opencontainers.image.revision=d7d0d9d854247d61f4018728e9a414cecf4fbb0e --label org.opencontainers.image.source=https://github.com/rancher/rancher.git --label org.opencontainers.image.url=https://github.com/rancher/rancher
```
[the remark this answers is not in the document]
Comment (rafaelbreno) [reply scripts/package]: Removed it, it shouldn't be hardcoded.

**Review by briandowns — APPROVED** [review]

**Review by igomez06 — DISMISSED** [review]
LGTM

**Comment by tashima42** [comment]
closing in favor of: https://github.com/rancher/rancher/pull/46205

**Pull request closed without merging**